prospero-cli/src/main/java/org/wildfly/prospero/cli/RepositoryDefinition.java

```diff
@@ -56,4 +69,31 @@ private static boolean isValidUrl(String text) {
             return false;
         }
     }
+
+    public static URI getAbsoluteFileURI(String text) throws ArgumentParsingException, URISyntaxException {
+        String path;
+        path = getPath(text);
```

Comment:
Why is the assignment on a separate line?

Reply:
I will fix this